### PR TITLE
Event Logging

### PR DESCRIPTION
**Pull request by Alroniks**
Related to https://oat-sa.atlassian.net/browse/TAO-2705
and also to https://oat-sa.atlassian.net/browse/TAO-2699

https://github.com/oat-sa/extension-tao-act/pull/271
https://github.com/oat-sa/tao-core/pull/881
https://github.com/oat-sa/extension-tao-eventlog/pull/6

### CODE REVIEW

File: views/js/controller/TaoEventLog/show.js [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+                    filterable: true
+                }],
+                
+                listeners: {
+                    /**
+                     * When a row is selected, we update the student viewer
+                     */
+                    selected: function selectRow(e, event) {
+
+                        console.log(e, event);
```
Comment (ssipasseuth) [inline views/js/controller/TaoEventLog/show.js]: console.log

File: controller/TaoEventLog.php [hunk 2]
```diff
@@ -42,11 +43,25 @@ public function __construct(){
      * A possible entry point to tao
      */
     public function index() {
-        echo __("Hello World");
+        $this->setData('author', 'Open Assessment Technologies SA');
```
Comment (siwane) [inline controller/TaoEventLog.php]: Could you remove this please

File: controller/TaoEventLog.php [hunk 3]
```diff
@@ -42,11 +43,25 @@ public function __construct(){
...
+        $this->setView('TaoEventLog/log.tpl');
     }
 
-    public function templateExample() {
-        $this->setData('author', 'Open Assessment Technologies SA');
-        $this->setView('TaoEventLog/templateExample.tpl');
+    /**
+     * Load json data with results
+     */
+    public function search()
```
Comment (siwane) [inline controller/TaoEventLog.php]: Could you check if parameters exist ?

> data, data.id, page, total

File: model/storage/RdsStorage.php [hunk 4]
```diff
@@ -0,0 +1,257 @@
...
+                self::USER_ID => $userIdentifier,
+                self::USER_ROLE => $userRole,
+                self::OCCURRED => $occurred,
+                self::PROPERTIES => json_encode($data)
+            ]
+        );
+
+        $id = $this->getPersistence()->lastInsertId(self::TABLE_NAME);
+
+        // todo clean data older than 90 days
```
Comment (siwane) [inline model/storage/RdsStorage.php]: This is a blocking factor, could you move this logic to 90 days. Discuss with Tom, it's mandatory

File: controller/TaoEventLog.php (hunk 3 above)
Comment (Alroniks) [inline controller/TaoEventLog.php]: All params will be checked in RdsStorage (they all are optional). It's controller for datatable component, so it uses always such way.

**Pull request merged**